### ISSUE TITLE
doctl auth init -t TOKEN should not print token to stdout

### ISSUE THREAD

**Issue by kc-dot-io** [digitalocean/doctl#1407]
**Replication Steps**

When using doctl the default behavior of token auth, is to print the token to stdout per `<redacted>`.
```
doctl auth init -t $TOKEN

Using token [<redacted>]

Validating token... OK
```

This behavior is undesirable as the default behavior because security. 

**Desired Behavior**
No tokens or sensitive information are sent to stdout.

**Actual Behavior**
A sensitive token is printed to stdout which can be inadvertedly leaked.

**Reasoning**
Security. When doctl is used in CI (or other scenarios), it results in tokens being written to stdout which is often stored in a log state, creating opportunity for tokens to be leaked, unccessarily.

It is a requirement to authenticate docker with the DO registry via doctl, so it's a critical use case.

If this DX intended to help users verify their token, maybe default off and flag on with `--verify`?

Thanks for the great work on the CLI. Happy to offer any further feedback that will help 🤘

**Comment by andrewsomething** [comment]
Hi @kc-dot-io 

What version of `doctl` are you using? As of PR https://github.com/digitalocean/doctl/pull/1337, which landed in [v1.92.0](https://github.com/digitalocean/doctl/releases/tag/v1.92.0), the token itself should not be printed to stdout.

**Comment by andrewsomething** [comment]
As mentioned above, this should no longer be the case in v1.92.0. If you're still seeing this behavior or I've misunderstood your report, please feel free to reopen.
